src/app/components/Title.module.css

```diff
@@ -0,0 +1,12 @@
+.title {
+  display: grid;
+  font-family: 'Bebas Neue', cursive;
```

Comment:
Wouldn't you have to import the font here again for it to be applied?
Or do you have it in your global.css?